docs/user-guide/00-introduction.md

```diff
@@ -27,26 +27,26 @@ Image Builder can create images in multiple output formats shown in the followin
-| QEMU Image    | qcow2     | .qcow2  | 
-| Disk Archive  | tar     | .tar      | 
-| Amazon Web Services | raw     | .raw      | 
-| Microsoft Azure | vhd     | .vhd      | 
-| Google Cloud Platform | gce    | .tar.gz     | 
-| VMware vSphere | vmdk     | .vmdk      | 
-| VMware vSphere | ova     | .ova     | 
-| Openstack | openstack     | .qcow2     | 
-| IoT Commit | iot-commit    | .tar      | 
-| IoT Container | iot-container | .tar      | 
-| IoT Installer | iot-installer | .iso     | 
-| IoT Raw Image| iot-raw-image  | .raw.xz      | 
-| IoT Simplified Installer | iot-simplified-installer | .iso      | 
-| IoT AMI | iot-ami | .ami      | 
-| IoT VMDK | iot-vmdk | .vmdk      | 
-| Installer | image-installer | iso     | 
-| Installer | live-installer | iso     | 
-| Oracle Cloud Infrastructure | oci     | .qcow2      | 
+| Name                              |         CLI name         | File extension | Description                                                                                 |  
+|-----------------------------------|:------------------------:|----------------|---------------------------------------------------------------------------------------------|  
+| QEMU Image                        |          qcow2           | `.qcow2`       | QCOW2 formatted disk image for use with QEMU/Libvirt.                                       | 
+| Disk Archive                      |           tar            | `.tar`         | Archive of the root filesystem without any bootloader or partition table.                   | 
+| Amazon Machine Image              |           ami            | `.ami`         | Disk image for use with AWS EC2.                                                            | 
+| Microsoft Azure                   |           vhd            | `.vhd`         | Disk image for use with Microsoft Azure.                                                    | 
+| Google Cloud Platform             |           gce            | `.tar.gz`      | Disk image for use with Google Cloud Platform.                                              | 
+| VMware vSphere                    |           vmdk           | `.vmdk`        | VMDK formatted disk image for use with VMware vSphere.                                      | 
+| VMware vSphere                    |           ova            | `.ova`         | OVA formatted disk image for use with VMware vSphere.                                       | 
+| Openstack                         |        openstack         | `.qcow2`       | QCOW2 formatted disk image for use with OpenStack.                                          | 
+| IoT Commit                        |        iot-commit        | `.tar`         | OSTree-based image to be used as a new OSTree commit.                                       | 
+| IoT Container                     |      iot-container       | `.tar`         | OSTree-based commit packaged within a container image for containerized deployments.        | 
```

Comment:
The IoT (and Edge) containers are actually little web server containers that serve an ostree commit.  It's meant to make it convenient to set up a single-commit ostree repo for building an ISO or disk image.

So maybe something like:
> Web server container that embeds and serves an ostree repository with a single commit.